modules/SugarFeed/Dashlets/SugarFeedDashlet/SugarFeedDashlet.php

```diff
@@ -596,7 +596,8 @@ public function getPostForm()
     {
         global $current_user;
 
-        if ((!empty($this->selectedCategories) && !in_array('UserFeed', $this->selectedCategories))
+
+        if ((!empty($this->selectedCategories) && !in_array('User Feed', $this->categories))
```

Comment:
Unnecessary parenthesis. We should also provide a third parameter for ```in_array``` to provide type safety.
```suggestion
        if (!empty($this->selectedCategories) && !in_array('User Feed', $this->categories, true)
```